go.mod

```diff
@@ -0,0 +1,8 @@
+module github.com/adofm/Maverick-Gliders.git
```

Comment:
This must reference the upstream repo, which is `YashKarthik2927`; not your fork.

Reply:
ok sir